src/components/TableBulkSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from 'react';
+
+import { BulkSelect, BulkSelectValue } from '@patternfly/react-component-groups';
+
+type TableBulkSelectProps = {
+  selectedIds: string[];
+  onSelect: (selectedIds: string[]) => void;
+  pageDataIds: string[];
+  dataIds: string[];
+};
+
+const TableBulkSelect: FC<TableBulkSelectProps> = ({
+  selectedIds = [],
+  pageDataIds = [],
```

Comment:
can you fix linting issues with files you've touched?

Reply:
(within reason)

Reply:
I've fixed several as a part of the latest commit, but changes to files like `StandardPage` that have nothing to do with this change are out of scope and bloat the PR.